raiden_contracts/constants.py

```diff
@@ -8,7 +8,7 @@
 from raiden_contracts.utils.type_aliases import ChainID
 
 # Do not change this, this is handled by bumpversion with .bumpversion_contracts.cfg
```

Comment:
Is this really still handled by bumpversion?

Reply:
No, and I was not seeing this comment.

Reply:
Fixed the comment.